code/src/App.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react'
 
+import Collection from 'components/Collection'
+
+
 export const App = () => {
+
   return (
     <div>
-      Find me in src/app.js!
+      <Collection />
     </div>
```

Comment:
Because there is nothing else happening in `App`, I feel like you could almost move everything from `Collection` into app to avoid somewhat redundant files. Not sure though, I'm no expert in structuring React.